bindings/py/nupic/bindings/regions/PyRegion.py

```diff
@@ -24,11 +24,28 @@
 """
 
 import numpy
+import collections
 
 RealNumpyDType = numpy.float32
 from abc import ABCMeta, abstractmethod
 
+class DictReadOnlyWrapper(collections.Mapping):
```

Comment:
Missed before merge but the two classes should have three blank lines preceding them (just to keep in mind)